### PR TITLE
279 QV Cleanup & NATSPEC

### PR DESCRIPTION
**Pull request by codenamejason**

### CODE REVIEW

File: contracts/strategies/qv-base/QVBaseStrategy.sol [hunk 1]
```diff
@@ -45,13 +60,51 @@ abstract contract QVBaseStrategy is BaseStrategy {
...
 
+    // slot 0
+    /// @notice The total number of votes cast for all recipients
+    uint256 public totalRecipientVotes;
+
+    // slot 1
+    /// @notice The number of votes required to review a recipient
+    uint256 public reviewThreshold;
+
+    // slot 2
```
Comment (0xKurt) [inline contracts/strategies/qv-base/QVBaseStrategy.sol:87]: love it, nice!

File: contracts/strategies/qv-simple/QVSimpleStrategy.sol [hunk 2]
```diff
@@ -40,8 +76,9 @@ contract QVSimpleStrategy is QVBaseStrategy {
     /// ===============================
 
     /// @notice Initialize the strategy
-    /// @param _poolId The pool id
-    /// @param _data The data
+    /// @dev Only `Allo` can call this function
+    /// @param _poolId The ID of the pool
+    /// @param _data The initialization data for the strategy
```
Comment (gabewin) [inline contracts/strategies/qv-simple/QVSimpleStrategy.sol:81]: If possible, can we add a custom with the data details here? Ex from RFP: /// @custom:data (uint256 voteThreshold, InitializeParams params)

File: contracts/strategies/qv-simple/QVSimpleStrategy.sol [hunk 3]
```diff
@@ -62,6 +100,7 @@ contract QVSimpleStrategy is QVBaseStrategy {
     }
 
     /// @notice Remove allocator
+    /// @dev Only the pool manager(s) can call this function
     /// @param _allocator The allocator address
```
Comment (gabewin) [inline contracts/strategies/qv-simple/QVSimpleStrategy.sol:104]: Add that this emits an ''AllocatorRemoved()' event to the @dev.

File: contracts/strategies/qv-simple/QVSimpleStrategy.sol [hunk 4]
```diff
@@ -54,6 +91,7 @@ contract QVSimpleStrategy is QVBaseStrategy {
     /// ====================================
 
     /// @notice Add allocator
+    /// @dev Only the pool manager(s) can call this function
```
Comment (gabewin) [inline contracts/strategies/qv-simple/QVSimpleStrategy.sol:94]: Add that this emits an ''AllocatorAdded()' event to the @dev.

**Review by gabewin — CHANGES_REQUESTED** [review]

**Review by 0xKurt — APPROVED** [review]

**Pull request merged**